### PR TITLE
Use tito for releases

### PR DESCRIPTION
**Pull request by xsuchy**

### CODE REVIEW

File: .github/workflows/check.yml [hunk 1]
```diff
@@ -32,13 +31,15 @@ jobs:
       - name: Configure build
         run: |
           chown -R builder. .
-          sudo -u builder -- ./autogen.sh
 
       - name: Build
-        run: sudo -u builder -- make --load-average=2
+        run: sudo -u builder -- tito buld --test --rpm
```
Comment (ernestask) [inline .github/workflows/check.yml:36]: There’s really no point in building things twice if you’re not storing the artifacts anywhere. Using Packit would solve the issue (and builds would happen in parallel and for multiple Fedora releases).

Comment (xsuchy) [reply .github/workflows/check.yml:36]: This should be the primary way of building. I put the other 'make' in `run test` so we can have the artifacts.

Comment (ernestask) [reply .github/workflows/check.yml:36]: I’m talking about the resulting RPMs.

Comment (ernestask) [reply .github/workflows/check.yml:36]: There’s also a typo in in that line.

**Comment by xsuchy** [comment]
updated

**Comment by xsuchy** [comment]
I added releasers for Fedora dist-git so we can do a release just with `tito release fedora-git`.

**Pull request merged**